src/features/pdf/PdfView2.tsx

```diff
@@ -84,6 +86,15 @@ export const PDFView2 = () => {
             .filter((node) => !pdfSettings?.excludedComponents.includes(node.item.id))
             .filter((node) => node.def.shouldRenderInAutomaticPDF(node as any))
             .map((node) => {
+              if (node.item.type === 'Summary2' && node.item.target?.taskId) {
+                return (
+                  <SummaryComponent2
+                    key={node.item.id}
+                    summaryNode={node as BaseLayoutNode<CompSummary2Internal, 'Summary2'>}
+                  />
+                );
+              }
+
```

Comment:
Btw, doesn't this mean that all other Summary2 components in a layout will fall through and render with ComponentSummary? Maybe that doesn't matter as `renderSummary2()` returns `null`. However, the comment over there (_"If the code ever ends up with a [...]"_) indicates that it shouldn't happen, but now it definitely will happen.